src/trace_processor/importers/ftrace/ftrace_parser.h

```diff
@@ -462,6 +468,10 @@ class FtraceParser {
   // Record number of transmitted bytes to the network interface card.
   std::unordered_map<std::string, uint64_t> nic_transmitted_bytes_;
 
+  // Record tid to migrate info.
+  std::unordered_map<UniqueTid, std::unordered_map<std::string, Variadic>>
```

Comment:
Creating a map of map of variadic is going to kill performance for no reason given all you are doing is storing three integers...

Please make a new struct for the three thigns you want to store and write to them. Then you can just read these in the end event.